Streetcode/Streetcode.BLL/HealthChecks/LivenessHealthChecks.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Streetcode.BLL.HealthChecks.MemoryMetrics;
+
+namespace Streetcode.BLL.HealthChecks
+{
+    public class LivenessHealthChecks : IHealthCheck
+    {
+        public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context, CancellationToken cancellationToken = default)
+        {
+            var client = new MemoryMetricsClient();
+            var metrics = client.GetMetrics();
+            var percentUsed = 100 * metrics.Used / metrics.Total;
+            var description = "System memory used is up to 80%";
```

Comment:
what about using constants, not magic strings? If I understand correctly, this message will be shown on UI. So we need to localize it. don`t we?